doc/zh-cn/admin-api.md

```diff
@@ -723,3 +723,15 @@ Content-Type: text/plain
 ```
 
 [Back to TOC](#目录)
+
+*地址*：/apisix/admin/plugins/meta_attributes
+
+*说明*: 插件元属性，包括插件的 `name`, `priority`, `priority`, `schema`, `consumer_schema` and `version`。
+
+> 请求方法:
+
+|Method   |请求 URI|请求 body|说明        |
+|---------|-------------------------|--|------|
+|GET      |/apisix/admin/plugins/meta_attributes/{plugin_name}|无|获取资源|
```

Comment:
I do not like this way(use a new API style)

We can add a new argument to the old API for this feature.

`/apisix/admin/schema/plugins/limit-count?type=priority`
`/apisix/admin/schema/plugins/limit-count?type=schema`
`/apisix/admin/schema/plugins/limit-count?type=consumer_schema `
`/apisix/admin/schema/plugins/limit-count?type=version`

we talked about this at Github issue or email list, but I can not find them :(

Reply:
i think to adopt a new API to return the meta attributes of all plugins, avoiding dashboard calls many times.
but perhaps the dashboard doesn't have to load all the plugin's meta properties at once, but only performs the loading when needed.

on the basis of the above example, if there is no type parameter, the default is to return the plugin's `type`,  `priority`, `schema`, `consumer_schema` and `version`. 
```
/apisix/admin/schema/plugins/limit-count

{
    "priority":
    "schema":
    "consumer_schema":
    "version":
    "type":
}
```
how ablot this?

Reply:
We cannot change the style of old API.  

 So we add new argument and return the different value.

Reply:
```
/apisix/admin/schema/plugins/limit-count?type=priority
```

When I see this API, I will think it is used to fetch the schema for the target plugin, but what is the type? Oh! it's used to fetch the metadata according to type 🤔 confused for me. We could use this way to compatible with the old API style now, but do we have any plan to improve it? more like a standard RESTful API?

Reply:
BTW,

```
/apisix/admin/plugins/{plugin_name}/meta_attributes
```

would be more RESTful.

We could use the old way to keep the style currently if possiable.

Reply:
> BTW,
> 
> ```
> /apisix/admin/plugins/{plugin_name}/meta_attributes
> ```
> 
> would be more RESTful.
> 
> We could use the old way to keep the style currently if possiable.

this style looks great!

Reply:
🤔 Emma then let's use the old way in 2.x? We could use the more RESTful API in breaking changes IMO, what do you think? @moonming @liuxiran @ShiningRush

Reply:
> ```
> 
> /apisix/admin/schema/plugins/limit-count?type=priority
> 
> ```
> 
> 
> 
> When I see this API, I will think it is used to fetch the schema for the target plugin, but what is the type? Oh! it's used to fetch the metadata according to type 🤔 confused for me. We could use this way to compatible with the old API style now, but do we have any plan to improve it? more like a standard RESTful API.

I don't like this style too.
We need restful API

Reply:
> 🤔 Emma then let's use the old way in 2.x? We could use the more RESTful API in breaking changes IMO, what do you think? @moonming @liuxiran @ShiningRush 

Metadata and schema are two different things, and should not be mixed in the same API

Reply:
Why not `/apisix/admin/plugins/{plugin_name}`? We don't need `meta_attributes` since we return everything from the plugin.

Reply:
Agree with metadata and schema are two different things, so for the schema, the old API could just remain the same.

and for the `meta_attributes`, IMHO, `GET /apisix/admin/plugins/{plugin_name}` is more simple😊

`GET /apisix/admin/plugins/{plugin_name}` for get {plugin_name}'s meta_attrs
`GET /apisix/admin/plugins` for get all plugins' meta_attrs

Reply:
> Why not `/apisix/admin/plugins/{plugin_name}`? We don't need `meta_attributes` since we return everything from the plugin.

This is also a good choice 👏

Reply:
> Why not `/apisix/admin/plugins/{plugin_name}`? We don't need `meta_attributes` since we return everything from the plugin.

This is also a good choice 👏

Reply:
🤔

```
GET /apisix/admin/plugins/{plugin_name}
```

This API aims to fetch that entity's data from my view point, could we return fields like schema, meta, etc?

Reply:
> > thinking Emma then let's use the old way in 2.x? We could use the more RESTful API in breaking changes IMO, what do you think? @moonming @liuxiran @ShiningRush
> 
> Metadata and schema are two different things, and should not be mixed in the same API

you are right, I was confusing metadate and schema.

Reply:
https://github.com/apache/apisix/blob/master/t/admin/plugins.t#L128
https://github.com/apache/apisix/blob/master/t/admin/plugins.t#L149

the APISIX API has done in this way for .

`/apisix/admin/plugins/basic-auth`
`/apisix/admin/plugins/basic-auth?schema_type=consumer`

and `/apisix/admin/plugins/{plugin-name}` is an old API for months. If we want to change it, it will not be backward compatible.

that is the original PR link: https://github.com/apache/apisix/pull/2509

Reply:
> thinking Emma then let's use the old way in 2.x? We could use the more RESTful API in breaking changes IMO, what do you think? @moonming @liuxiran @ShiningRush

If we want to do in this way, need to discuss in mail-list at first:

![image](https://user-images.githubusercontent.com/6814606/100580728-72545980-3321-11eb-93e0-bd5dc54f2868.png)